src/main/java/frc/team2412/robot/commands/launcher/AimTowardsSpeakerCommand.java

```diff
@@ -75,6 +70,10 @@ public void end(boolean interrupted) {
 
 	@Override
 	public boolean isFinished() {
-		return true;
+		return (MathUtil.isNear(
+						drivebaseSubsystem.getPose().getRotation().getDegrees(),
+						yawTarget.getDegrees(),
+						HEADING_TOLERANCE)
+				&& launcherSubsystem.isAtSpeed());
```

Comment:
Or should this be checking `launcherSubsystem.isAtAngle(...)`, since the angle target of the launcher is being set but this doesn't wait for it to be reached.